packages/expo-video/ios/VideoPlayer.swift

```diff
@@ -1,11 +1,29 @@
 // Copyright 2023-present 650 Industries. All rights reserved.
 
 import AVFoundation
+import MediaPlayer
 import ExpoModulesCore
 
 internal final class VideoPlayer: SharedRef<AVPlayer>, Hashable {
   lazy var contentKeyManager = ContentKeyManager()
 
+  var isLooping = false {
+    didSet {
+      self.applyIsLooping()
```

Comment:
```suggestion
     applyIsLooping()
```
Can we remove the `self` from here?